src/js/grapl-cdk/lib/historydb.ts

```diff
@@ -125,7 +109,6 @@ export class HistoryDb extends cdk.Construct {
         );
         this.ip_connection_history.grantReadWriteData(service.event_handler);
         this.asset_history.grantReadWriteData(service.event_handler);
```

Comment:
We can remove all of these tables like 'network connection' etc. They are unused, as we are now general over Static and Session identified nodes.